libs/logging/src/log_event_ids.ts

```diff
@@ -1175,43 +1178,47 @@ const UnknownError: LogDetails = {
 export function getDetailsForEventId(eventId: LogEventId): LogDetails {
   switch (eventId) {
     case LogEventId.ElectionConfigured:
-      return ElectionConfiguredEvent;
```

Comment:
The variable name of type `LogDetail` is private to this file and now autogenerated, so the enum member (eg. L1177) and variable name (eg. L1178) can no longer diverge.